src/OpenTelemetry.Instrumentation.EventCounters/EventCountersInstrumentationOptions.cs

```diff
@@ -0,0 +1,57 @@
+// <copyright file="EventCountersInstrumentationOptions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+
+namespace OpenTelemetry.Instrumentation.EventCounters;
+
+/// <summary>
+/// EventCounterListener Options.
+/// </summary>
+public class EventCountersInstrumentationOptions
+{
+    private readonly HashSet<string> eventSourceNames = new();
+
+    /// <summary>
+    /// Gets or sets the subscription interval in seconds.
+    /// </summary>
+    public int RefreshIntervalSecs { get; set; } = 1;
+
+    /// <summary>
+    /// Listens to EventCounters from the given EventSource name.
+    /// </summary>
+    /// <param name="eventSourceName">The EventSource name.</param>
+    public void AddEventSource(string eventSourceName)
```

Comment:
Use `AddSource(params string[] names)` instead?

Reply:
The user code can loop through their array of names and call AddEventSource multiple times, I'd rather keep it simple

Reply:
As opposed to just passing their array to `AddEventSource` method? I think accepting `params` makes it simpler.

Refer to `AddSource` and `AddMeter` methods of the SDK:
- https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry/Trace/Builder/TracerProviderBuilderBase.cs#L84
- https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry/Metrics/Builder/MeterProviderBuilderBase.cs#L83

Reply:
Ah, if there's a precedence of this then I will make that suggested change :) wasn't aware if that's a common practice for either C# or OTel .NET